mock/truffle.js

```diff
@@ -3,10 +3,20 @@ module.exports = {
     development: {
       host: 'localhost',
       port: 8545,
-      network_id: '*' // Match any network id
+      network_id: '*',
+      websockets: true
     }
   },
   mocha: {
     reporter: 'eth-gas-reporter',
+    reporterOptions: {
+      currency: "chf",
+      gasPrice: 21,
+      onlyCalledMethods: true,
+      noColors: true,
+      rst: true,
+      rstTitle: 'Gas Usage',
+      showTimeSpent: true
+    }
   }
```

Comment:
Sorry, the tests here are slightly insane - there are actually two truffle.js 's and two visual inspections. One is `regular` (this one) and one is for options and is a template file that gets swapped in by a shell script. I think this file can be left in original state because your work doesn't affect the options. Also guessing this was done by the shell script somehow, during a crash or something. . . .

Reply:
Yes that is most likely it.  I noticed that in my git add, but figured it was something the shell script added intentionally